docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -339,15 +339,20 @@ post:
       is: [ secured, deployable ]
       queryParameters:
         host:
-          description: all tasks of that application on the supplied slave are killed
+          description: all tasks of that application on the supplied slave are killed.
         scale:
           type: boolean
           description: If `scale=true` is specified, then the application is scaled down by the number of killed tasks. Only possible if `wipe=false` or not specified.
           default: false
         wipe:
           type: boolean
           description: If `wipe=true` is specified and the app uses local persistent volumes, associated dynamic reservations will be unreserved, and persistent volumes will be destroyed. Only possible if `scale=false` or not specified.
-          default: false
+        killCount:
+          type: integer
+          description: The number of tasks to kill. A negative value signifies killing all running tasks.
+        noOp:
```

Comment:
Marathon groups API calls this parameter `dryRun`. Please use the same name here as well.